setup.py

```diff
@@ -14,7 +14,14 @@
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
     ],
-    install_requires=['pytest', 'virtualenv'],
+    install_requires=[
+        'pytest',
+        # RequirementParseError got refactored in v49
+        # (https://github.com/pypa/setuptools/blob/main/CHANGES.rst#changes-20])
+        # and doesn't appear to be entirely backwards compatible.
+        'setuptools>18.5,<49',
```

Comment:
this should get fixed instead of constraining -- constraining setuptools doesn't actually work out all that well (and pulls in an unnecessary dependency)

Reply:
https://github.com/Yelp/requirements-tools/issues/54

Reply:
this should be done before merging this, please

Reply:
You're welcome to submit a pull request. The goal of this PR is to migrate to github actions and get a working build.

Reply:
please do not regress this.  this breaks the packaging

Reply:
I'll drop this change and leave the build broken if this breaks a critical workflow. In the future, please provide a bit more insight as to your reasoning. "This should be done before merging" does not explain your reasons sufficiently. Thanks!

Reply:
or you could take the two seconds it would have taken to fix the actual problem instead?   you've spent way more time on this than it would have taken to just fix the problem

Reply:
It may have taken you 2 seconds to fix the problem. Please do not presume to know how long it would take anyone else.

Reply:
you already found the answer to fixing it though!  the type moved, simply use the new type?